inductiva/tasks/run_simulation.py

```diff
@@ -57,6 +57,11 @@ def run_simulation(
     if computational_resources is not None:
         logging.info("Task %s submitted to the queue of the %s.", task_id,
                      computational_resources)
+
+        if not computational_resources.is_running():
+            logging.warning(
```

Comment:
I agree with Hugo. Also, there's no need to continue with the flow because we should get an error if the resource is not running. In any case, I've asked for a new ticket to clarify how the backend handles these cases because it is not clear what the behaviour is when the resource has not yet been started vs having already been terminated.

Reply:
So the problem has been solved in the backend and this pull request is no longer necessary, correct?

Reply:
Any update or suggestions for the error handling?